src/docs/product/data-management-settings/event-grouping/index.mdx

```diff
@@ -5,23 +5,28 @@ redirect_from:
   - /data-management/rollups/
   - /learn/rollups/
   - /data-management/event-grouping/
-description: "Learn about fingerprints, the grouping algorithm, grouping by stack trace, grouping by exceptions, and fallback grouping."
+  - /guides/grouping-and-fingerprints/
+  - /product/sentry-basics/guides/grouping-and-fingerprints/
+description: "Learn about fingerprints, the grouping algorithm, grouping by stack trace, grouping by exceptions, and fallback grouping. Learn more about how Sentry groups issues together as well as different approaches for updating how events group into issues"
 ---
 
-All events have a fingerprint. Events with the same fingerprint are grouped together into an issue.
+A fingerprint is a way to uniquely identify an event, and all events have one. Events with the same fingerprint are grouped together into an issue. So if two events have the same stack trace, then they will group into one issue.
 
-By default, Sentry will run one of our built-in grouping algorithms to generate a fingerprint based on information available within the event such as `stacktrace`, `exception`, and `message`. To extend the default grouping behavior or change it completely, you can use a combination of the following options:
+By default, Sentry will run one of our built-in grouping algorithms to generate an event fingerprint based on information available within the event, such as `stacktrace`, `exception`, and `message`. If the way that Sentry is grouping issues for your organization is already ideal, you don't need to make any changes. However, you can extend the default grouping behavior or change it completely, using a combination of the following options:
 
 1. In your SDK, using [SDK Fingerprinting](/platform-redirect/?next=/usage/sdk-fingerprinting/). Note that is not supported in WebAssembly.
 2. In your project, using [Fingerprint Rules](./fingerprint-rules/)
 3. In your project, using [Stack Trace Rules](./stack-trace-rules/)
+4. In your project, using [Grouping Breakdown](./grouping-breakdown). Early Adopters only.
```

Comment:
The current Grouping Breakdown only provides information, it does not allow the user to extend or change the grouping behavior. It probably will in the future. @untitaker what do you think, should we list it here anyway?